fh-mbaas.js

```diff
@@ -102,6 +102,8 @@ function loadConfig(cb) {
   // read our config file
   var configFile = process.env.conf_file || args._[0];
 
+  console.log(fhconfig);
+
   fhconfig.init(configFile, requiredvalidation, function(err) {
```

Comment:
Shows that @matskiv has reason the name of the function reload was changed in the 2.0.0 of the dep fh-config. Also, shows that for we move forward with it instead of you use the init here we could use the reloadRawConfig.